.github/workflows/reusable_on_release.yaml

```diff
@@ -0,0 +1,122 @@
+name: Release pipeline
+
+# Example usage from the caller workflow:
+#
+# It requires the following secrets (inheriting them is recommended for simplicity)
```

Comment:
nit: this is described below in the secret section , i would remove this part of the comment listing the required secrets